src/styles/blocks/star.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 12px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0 0 0 18px;
+  }
+}
+
+.stars--1 > .stars__star:nth-child(-n+1),
+.stars--2 > .stars__star:nth-child(-n+2),
+.stars--3 > .stars__star:nth-child(-n+3),
+.stars--4 > .stars__star:nth-child(-n+4),
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
Need to fix with checklist
<img width="349" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/0a1315c1-0c2c-4d9b-a86f-07ba2031737f">

Reply:
not fixed, use `@for` loop